plugin/test/unit/helpers/draft.config.spec.ts

```diff
@@ -0,0 +1,198 @@
+/* eslint-disable  @typescript-eslint/no-explicit-any */
+import { resolve, join } from 'path'
+
+import {
+  copySync,
+  ensureFileSync,
+  readdir,
+  readJSON,
+  writeJSON,
+} from 'fs-extra'
+import { v4 as uuidv4 } from 'uuid'
+
+import { createMetadataFileAndCopyDatastore } from '../../../src/helpers/config'
+
+jest.mock('fs-extra')
+jest.mock('uuid')
+
+// TODO: Need proper TS types here!
```

Comment:
I know this is awful! We should set the correct types of the mocked functions.

Reply:
@estephinson showed me how to improve the TS types here as I'm not overly familiar with TypeScript right now. I'll leave it as an exercise for whoever picks this up though as there's some fun with the types when adding the mocked data to the `readdir` mock. 😉 

```
const mockReadJSON = readJSON as jest.MockedFunction<typeof readJSON>
const mockEnsureFileSync = ensureFileSync as jest.MockedFunction<typeof ensureFileSync>
const mockCopySync = copySync as jest.MockedFunction<typeof copySync>
const mockWriteJSON = writeJSON as jest.MockedFunction<typeof writeJSON>
const mockReaddir = readdir as jest.MockedFunction<typeof readdir>
const mockUuidv4 = uuidv4 as jest.MockedFunction<typeof uuidv4>
```